assets/scss/modules/_backgrounds.scss

```diff
@@ -0,0 +1,68 @@
+.background {
+  background-repeat: no-repeat;
+  background-size: contain;
+  position: absolute;
+  z-index: -1;
+  top: 50%;
+  left: 50%;
+
+  &.bg-a {
+    background-image: url("../img/background-pattern-a.svg");
+    height: 653px;
+    width: 1551px;
+  }
+
+  &.bg-b {
+    background-image: url("../img/background-pattern-b.svg");
+    height: 270px;
+    width: 230px;
+  }
+
+  &.bg-c {
+    background-image: url("../img/background-pattern-c.svg");
+    height: 227px;
+    width: 289px;
+  }
+
+  &.bg-d {
```

Comment:
Why are we including backgrounds that we do not use? Where did so many backgrounds come from?

Reply:
These backgrounds are used in the new site, it seemed a good idea to add them to the library and it may be that in the future they will be used...

Reply:
I'm not sure, we should have only what we need. They have to be all common elements. The backgrounds are momentary look and fell. If in 6 months the institutional image changes, this does not work for us anymore. Sites has its own graphic style very different from the library.

Reply:
Creo que está bueno agregarlas, pero que me parece que un PR relacionado con la página de confirmación no es el mejor lugar para hacerlo.

Por otro lado, estaría bueno que si están en la librería, haya forma de acceder a esas imágnenes desde los HTML de demostración.

@IgnacioRodrigues te parece quitarlas de este PR y hacer uno nuevo con las imágenes y con la demo de ellas?

Reply:
> I'm not sure, we should have only what we need. They have to be all common elements. The backgrounds are momentary look and fell. If in 6 months the institutional image changes, this does not work for us anymore. Sites has its own graphic style very different from the library.

Perdón, no había visto eso. Y estoy de acuerdo, creo que es buena idea dejar las imágenes en la webapp.

---

Nota poco importante:

> Sites has its own graphic style very different

No estoy de acuerdo con la frase literal. Esta no es una librería agnostica como bootstrap, es una librería de _estilos_ y de marca, osea, es muy probable que los estilos de los gráficos tengan mucho en común entre los sistemas que usen la librería. 

Pero es cierto que los sistemas que usen la librería pueden tener imágenes muy distintas aunque todas tengan el mismo estilo.

Reply:
Uhh, entendí todo mal, disculpen el ruido. Aunque creo que aplica casi todo lo que dije antes.

Ahora se cuales son las imágenes, no me parece mal que a la larga tengamos una libería de _chirimbolos_ en la librería, que cada app/sitio los pueda usar donde quiera.

También me gustaría que no haya estilos específicos para login, signup, reports o lo que sea en la librería.

Pero, entiendo que ahora es algo práctico. Entonces, intententemos ser minimalistas y solo meter lo mínimo necesario.

Reply:
Se entendió todo @andresmoschini y @GusBaamonde, concuerdo en tener solo lo necesario en la libreria y luego una ves que este mas cerrado lo que necesitamos. Armar alguna libreria de chirimbolos (a futuro). Gracias chicos!